backend/.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,20 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Can you add to gitignore the .idea specific files?

Reply:
yes of course

Reply:
added

Reply:
Looks like the files are still pushed in the commit?

Reply:
Now they should be removed...